.github/workflows/zeebe-ci.yml

```diff
@@ -520,3 +523,4 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
+          user_description: "Core Features"
```

Comment:
❓ Failure to auto merge a PR is that core features? I'd argue that this is more general CI.